apigee/config/config.proto

```diff
@@ -31,4 +31,6 @@ message Params {
     string env_name      = 4;
     string key           = 5;
     string secret        = 6;
+
+    bool use_uap = 7;
```

Comment:
As mentioned above, we won't keep both impls... and definitely won't want to let customers decide on this, but this is fine for testing now.

Reply:
OK, added a comment to let people know that the feature is experimental.

Reply:
I wasn't clear. I meant that there should be no reason to have this field in the config or the old implementation at all once we've tested and merged this.